modules/account-lib/src/coin/sol/utils.ts

```diff
@@ -54,6 +54,7 @@ export function isValidPrivateKey(prvKey: string | Uint8Array): boolean {
 
 /** @inheritdoc */
 export function isValidPublicKey(pubKey: string): boolean {
```

Comment:
Param should be optional if it can be undefined. Otherwise callers should be fixed as they should get a compile error if they attempt to pass a value which could be undefined

Reply:
The caller (in this case is the IMS and is in JS) 
PR for reference https://github.com/BitGo/indexerdb-microservice/pull/510
@noel-bitgo

Reply:
I makes no sense to use optional when its the value we want to validate in an isValid method